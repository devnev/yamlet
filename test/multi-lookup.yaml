locals:
  a:
    b:
      c: 1
  d:
    - - 1
      - 2
result:
  map: (a.b.c)
  seq: (d[0][1])
