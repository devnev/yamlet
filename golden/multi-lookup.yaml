map: 1
seq: 2
